### PR TITLE
Make welcome msg join url configurable

### PR DESCRIPTION
**Pull request by kargakis**
/cc stevekuznetsov MHBauer spiffxp

### CODE REVIEW

**Comment by kargakis** [comment]
Opened https://github.com/kubernetes/test-infra/pull/5952 for the config update.

File: prow/plugins/trigger/trigger.go [hunk 1]
```diff
@@ -99,7 +99,8 @@ func getClient(pc plugins.PluginClient) client {
 
 func handlePullRequest(pc plugins.PluginClient, pr github.PullRequestEvent) error {
 	trustedOrg := trustedOrgForRepo(pc.PluginConfig, pr.Repo.Owner.Login, pr.Repo.Name)
-	return handlePR(getClient(pc), trustedOrg, pr)
+	joinOrgURL := pc.PluginConfig.TriggerFor(pr.Repo.Owner.Login, pr.Repo.Name).JoinOrgURL
```
Comment (cjwagner) [inline prow/plugins/trigger/trigger.go]: `TriggerFor` can return nil. This will segfault for orgs that don't explicitly specify a trustedOrg.

Comment (kargakis) [reply prow/plugins/trigger/trigger.go]: Opened https://github.com/kubernetes/test-infra/pull/5978.

**Pull request merged**